apps/i18n/aichat/en_us.json

```diff
@@ -1,12 +1,95 @@
 {
   "aboutAichatLab": "About AI Chat Lab",
+  "addCustomizationButtonText": "Add",
+  "addCustomizationCompletionText": "Added",
   "aichatWorkspaceHeader": "AI Chat",
+  "aichatView_screenReader_EditModeButton": "Edit Mode",
+  "aichatView_screenReader_UserViewModeButton": "User View Mode",
+  "aiCustomizations_modelCardInfo": "Model card information",
+  "aiCustomizations_retrieval": "Retrieval",
+  "aiCustomizations_selectedModel": "Selected model",
+  "aiCustomizations_systemPrompt": "System Prompt",
+  "aiCustomizations_temperature": "Temperature",
   "chatComponentsHeader": "AI Chat Components",
+  "chatEventDescriptions_copyChat": "The user copied the chat history.",
+  "chatEventDescriptions_clearChat": "The user cleared the chat workspace.",
+  "chatEventDescriptions_loadLevel": "The user loaded the aichat level.",
+  "chatEventDescriptions_waitForResponse": "Waiting for response",
+  "clearChatButtonText": "Clear chat",
   "continue": "Continue",
+  "copyChatButtonText": "Copy chat",
+  "copyChatContainsProfanity": "[FLAGGED AS PROFANITY]",
```

Comment:
Question for @mgc1194 -  These `[` and `]`s in these strings are not intended to be markdown. Is it a problem to have these characters in the string?

Reply:
or @castro-jorge

Reply:
Hello Cassi, it is not a problem to have these bracketed variables in strings. They are normally there in place of links. I have a question, what is the behavior for which this string is being used? Is the string used to mask the chat containing profanity? Do users get to see the "FLAGGED AS PROFANITY" text?

Reply:
Yes, when you copy the chat history in the chatbot window, we format a chat log on your clipboard. If the message was caught by our safety layer, we'll replace the message text with this string.